### ISSUE TITLE
Files can't be saved when using php-cs-fixer automatically on save

### ISSUE THREAD

**Issue by blizzardengle** [sdobreff/vscode-php-resolver#5]
When saving a PHP file with `editor.formatOnSave` enabled, `php-cs-fixer` runs and fixes the issues it can, but then places the changes into the file after the initial save command. This means any PHP files that still contain errors will show in the editor as an unsaved file.

The only way to save the file as is, is to either close the file and choose save in the dialog box that appears, or use the command pallet to save the file without formatting; both of which trigger an error that is placed in the output console. I think the proper way to handle this would be to sutomatically run `php-cs-fixer`, place the resulting code in the file (which is already being done), and then automatically trigger save without formatting so the file does not get stuck in a reformatting loop.

PS: Does your extension or vscode prompt (notify) `All fixable errors were resolved` when saving? Could that message be disabled or removed?

**Comment by blizzardengle** [comment]
This is the error that is shown if you close the file and click save in the dialog box:

> phpcs - Failed collecting proper output - Cannot read properties of undefined (reading 'document') [... lots of output ...]

I think this is caused by `vscode-php-resolver` looking to format the file as part of the on save event but it was also just closed so is no longer open in vscode (the document).

**Comment by tanc** [comment]
Agreed, this extension doesn't seem to work properly when set as the default formatter for PHP files. It correctly formats but then is always stuck in a modified state. If there was a linting error that was corrected on save the error is still shown.

**Comment by tanc** [comment]
I just re-installed this extension but the problem remains. @sdobreff any chance of a fix?
